projects/demo/src/modules/components/input-date-time/input-date-time.component.ts

```diff
@@ -62,25 +62,27 @@ export class ExampleTuiInputDateTimeComponent extends AbstractExampleTuiControl
         HTML: import('./examples/5/index.html?raw'),
     };
 
-    readonly minVariants: ReadonlyArray<TuiDay | [TuiDay, TuiTime]> = [
+    readonly minVariants: ReadonlyArray<TuiDay | [TuiDay, TuiTime] | null> = [
+        null,
         TUI_FIRST_DAY,
         new TuiDay(2017, 2, 5),
         new TuiDay(1900, 0, 1),
         new TuiDay(new Date().getFullYear() + 3, 1, 1),
         [this.today.append({day: -1}), new TuiTime(12, 20)],
     ];
 
-    min: TuiDay | [TuiDay, TuiTime] = this.minVariants[0];
+    min: TuiDay | [TuiDay, TuiTime] | null = this.minVariants[0];
 
-    readonly maxVariants: ReadonlyArray<TuiDay | [TuiDay, TuiTime]> = [
+    readonly maxVariants: ReadonlyArray<TuiDay | [TuiDay, TuiTime] | null> = [
+        null,
```

Comment:
null should not be in the list, it is lack of value, not a particular value. Demo portal adds cleaner to select if null is in the type automatically.